lib/collection.js

```diff
@@ -727,7 +727,8 @@ Collection.prototype.insert = deprecate(function(docs, options, callback) {
  * @param {number} [options.wtimeout] The write concern timeout.
  * @param {boolean} [options.j=false] Specify a journal write concern.
  * @param {boolean} [options.bypassDocumentValidation=false] Allow driver to bypass schema validation in MongoDB 3.2 or higher.
- * @param {Array} [options.arrayFilters] optional list of array filters referenced in filtered positional operators
+ * @param {array|object} [options.arrayFilters] optional list of array filters referenced in filtered positional operators
```

Comment:
The documentation doesn't seem updated to reflect the addition of a supported `object` type

Reply:
Good catch. I'll remove that.